miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java

```diff
@@ -139,6 +139,7 @@
 import uk.ac.bbsrc.tgac.miso.webapp.util.BulkEditTableBackend;
 import uk.ac.bbsrc.tgac.miso.webapp.util.BulkPropagateTableBackend;
 
+@SuppressWarnings("squid:S3776")
```

Comment:
Would rather not have this here, as a couple of the methods are pretty ugly and it's good that Sonar complains about it